### PR TITLE
Run hugo via `go run` and lock its version

### PR DESCRIPTION
**Pull request by silverwind**
- Don't rely on obscure docker images like `plugins/hugo`
- Lock down `hugo` to same version the image had used
- Remove unnecessary verbosity in `trans-copy`
- Rename `trans-copy` to `trans-copy.sh`

### CODE REVIEW

**Comment by silverwind** [comment]
Note, this does upgrade hugo from `v0.70.0` to `v0.108.0` as apparently the image hasn't upgraded it since over two years.

**Comment by silverwind** [comment]
Locking to same hugo version as I'm not motivated to find out what hugo broke and docs are not viewable locally for some reason (they redirect to https://docs.gitea.io but even when I remove the redirect, nothing renders).

File: .drone.yml [hunk 1]
```diff
@@ -928,13 +928,10 @@ trigger:
...
   - name: build-docs
-    image: plugins/hugo:latest
-    pull: always
+    image: golang:1.19
     commands:
-      # https://github.com/drone-plugins/drone-hugo/issues/36
-      - apk upgrade --no-cache libcurl && apk add --no-cache make bash curl
       - cd docs
-      - make trans-copy clean build
+      - cd docs && make trans-copy clean build
```
Comment (techknowlogick) [inline .drone.yml]:
```suggestion
      - make trans-copy clean build
```
the step above cd's into docs

Comment (silverwind) [reply .drone.yml]: Ah I missed that. Should probably move these targets into the main Makfile so cd'ing around becomes unneccesary, but I'm deferring that for later.

**Review by techknowlogick — APPROVED** [review]

**Comment by jolheiser** [comment]
We will need to find a version of Hugo without any [replace directives](https://github.com/gohugoio/hugo/blob/7f47b99ea9a7ba7759c4f9424fedd4591e6da497/go.mod#L72) in the `go.mod`, otherwise we cannot directly `go run` it.

File: docs/Makefile [hunk 2]
```diff
@@ -2,6 +2,8 @@ THEME := themes/gitea
 PUBLIC := public
 ARCHIVE := https://dl.gitea.io/theme/master.tar.gz
 
+HUGO_PACKAGE := github.com/gohugoio/hugo@v0.70.0
```
Comment (jolheiser) [inline docs/Makefile]:
```suggestion
HUGO_PACKAGE := github.com/gohugoio/hugo@v0.81.0
```
This appears to be the first version with a clean (no replace directives) module.
https://github.com/gohugoio/hugo/blob/v0.80.0/go.mod
vs
https://github.com/gohugoio/hugo/blob/v0.81.0/go.mod

I haven't checked if this version will work, just noted that it's the minimum viable for `go run`

**Review by lunny — APPROVED** [review]

**Comment by silverwind** [comment]
There is a checksum mismatch in https://drone.gitea.io/go-gitea/gitea/64798/4/5. Wonder how that is even possible, is sum.golang.org broken?

```
go run github.com/gohugoio/hugo@v0.81.0 --baseURL="/" --cleanDestinationDir
go: downloading github.com/gohugoio/hugo v0.81.0
go: github.com/gohugoio/hugo@v0.81.0: github.com/gohugoio/hugo@v0.81.0: verifying module: checksum mismatch

	downloaded: h1:zToI15wYh6oh4SRAaoy39XixwmiPB0DZDjsgu9szknY=
	sum.golang.org: h1:PX8TYe1nrWOOMBWtQ/YvKs6QRrOjC5/RrZGE4tBb6EE=

SECURITY ERROR
This download does NOT match the one reported by the checksum server.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.

**Comment by silverwind** [comment]
Might have something to do with GOPROXY, these fail:

```
GOPROXY=https://goproxy.io go run github.com/gohugoio/hugo@v0.81.0 --help
GOPROXY=direct go run github.com/gohugoio/hugo@v0.81.0 --help
```

This works:

```
go run github.com/gohugoio/hugo@v0.81.0 --help
```

**Pull request merged**